doc/classes/@GlobalScope.xml

```diff
@@ -161,9 +162,9 @@
 			<description>
 				Returns the cosine of angle [code]angle_rad[/code] in radians.
 				[codeblock]
-				# Prints 1 then -1
-				print(cos(PI * 2))
-				print(cos(PI))
+				cos(PI * 2)     # Returns 1.0
+				cos(PI)         # Returns -1.0
+				cos(deg2rad(90)) # Returns 0.0
```

Comment:
If the goal was to have alignment, the comments are not aligned. Although we could also just not have alignment and have each comment preceded by one space (I'm not sure which style is desired here).